backup.rb

```diff
@@ -66,41 +71,56 @@ def get_images!
     @boards.each do |board|
       print "[#{board[:name]}] ".green + "found "
       visit PINTEREST + board[:path]
-      load_all_content!
-      board[:image_urls] = js <<-JS
-        $('.pin').map(function(){ return $(this).data('closeup-url') });
-      JS
+      sleep 3
```

Comment:
by testing came to conclusion that need sleep to page get fully loaded before js evaluation